cypress/support/commands.js

```diff
@@ -176,11 +173,11 @@ Cypress.Commands.add('postMessageReplyInRHS', (message) => {
 });
 
 Cypress.Commands.add('getLastPost', () => {
-    return cy.get('#postListContent').children().last();
+    return cy.get('#postListContent').children().children().children().children().children().first();
 });
 
 Cypress.Commands.add('getLastPostId', () => {
-    return cy.get('#postListContent').children().last().invoke('attr', 'id').then((divPostId) => {
+    return cy.get('#postListContent').children().children().children().children().children().first().invoke('attr', 'id').then((divPostId) => {
```

Comment:
Deeply nested access to children of postListContent can't be avoid due to the external components (`AutoSizer `, `DynamicSizeList `) introduced recently by [virtualized post list](https://github.com/mattermost/mattermost-webapp/blob/ef76bf9161e665792cb77277ff1fbee9c7dd1aeb/components/post_view/post_list.jsx#L465).

Reply:
```suggestion
    return cy.get('#postListContent [id^=post]:first').invoke('attr', 'id').then((divPostId) => {
```

Reply:
That's sweet, looks good to me, thanks @cometkim!